Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM centos:7
+
+RUN yum update -y && \
+    yum install -y \
+    epel-release-7 \
+    zip-3.0 \
```

Comment:
Again, can we just specify `zip` instead of `zip-3.0`?

Reply:
Done. Also did the same for unzip, maven and make for consistency.